tasks/publisher.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Include initial OS-specific tasks
+  ansible.builtin.include_tasks: init_{{ ansible_os_family | lower }}.yml
+  vars:
+    _cvmfs_role: publisher
+    _cvmfs_upgrade: "{{ cvmfs_upgrade_server }}"
+
+- name: Include Apache tasks
+  ansible.builtin.include_tasks: apache.yml
+
```

Comment:
What is the purpose of this include for the publishers? I don't see apache requirement in https://cvmfs.readthedocs.io/en/stable/cpt-repository-gateway.html

Reply:
yes I just checked in all the docs including the official document it does not have an http package so I will delete it

Reply:
fixed